poi-ooxml/src/main/java/org/apache/poi/xwpf/usermodel/XWPFRun.java

```diff
@@ -1447,6 +1566,19 @@ public enum FontCharRange {
         hAnsi /* high ansi */
     }
 
+    /**
+     * @see <a href="https://www.ecma-international.org/publications-and-standards/standards/ecma-376/">OOXML Spec</a>
```

Comment:
Can you pull out the text? Or tell us exactly where to find these sections. The ECMA specs are an insane mess with zips inside zips. 4 separate parts.

This enum should be its own class but the question is whether it is docx specific or needs to appear in a more general package.

Reply:
The sections doesn't mention what a complex script is because it's something related to the typesetting. However, these sections are titled:
- 17.3.2.2 bCs (Complex Script Bold)
- 17.3.2.17 iCs (Complex Script Italics)
- 17.3.2.38 sz (Non-Complex Script Font Size)
- 17.3.2.39 szCs (Complex Script Font Size)

and in them they specify that these properties must have an explicit tag with it's value for them to work with complex scripts. I'm not sure if i should include the entire paragraphs or only a link to the PDF zip file? (it's 1 file only).

https://www.ecma-international.org/wp-content/uploads/ECMA-376-1_5th_edition_december_2016.zip

What would you suggest?